README.rst

```diff
@@ -1,130 +1,165 @@
+.. code-block:: bash
 
-Read more at our `documentation <http://coala.rtfd.org/>`__.
+    $ pip3 install coala-bears
 
-Why use coala?
---------------
+To install the latest development version run:
 
-- *coala* provides **built-in checking routines** (named bears in *coala*).
-- **Serves your requirement**: You can easily write your own checks (using
-  bears).
-- *coala* provides **user-friendly interfaces** like json, formatted and
-  interactive output in the CLI and *plugins for various editors* are
-  available as well.
-- **Optimal performance**: *coala* manages parallelizing the checking-routines
-  without you having to worry.
-- **Unified interface**: One tool for all programming languages.
+.. code-block:: bash
 
-If you want to learn more about *coala*, its functionality and its usage,
-please take a look at our
-`tutorial <http://coala.rtfd.org/en/latest/Users/Tutorials/Tutorial.html>`__.
+    $ pip3 install coala-bears --pre
 
-Installation
-------------
+|PyPI| |Windows| |Linux|
 
-To install *coala* with the official set of analysis routines you can simply run
-``pip3 install coala-bears``.
+-----
 
-*coala* only, without the official bears, can be installed with
-``pip3 install coala``. If you need more information about the installation and
-dependencies, take a look at our `installation documentation
-<http://coala.rtfd.org/en/latest/Users/Install.html>`__.
+=====
+Usage
+=====
 
-The latest code from master is automatically deployed to PyPI as a
-development version. Get it with ``pip3 install coala --pre``.
+To get the list of arguments and their meaning, run:
 
-|PyPI|
+.. code-block:: bash
 
-Authors
--------
+    $ coala --help
 
-*coala* is maintained by a growing community. Please take a look at the
-meta information in `setup.py <setup.py>`__ for current maintainers.
+To run coala over your project, create a ``.coafile``, the project-specific
+configuration file that will store all your settings for coala. A sample
+``.coafile`` will look something like this:
 
-Project Status
---------------
+.. code-block:: bash
 
-|Linux Build Status| |Windows Build status|
+    [Default]
+    files = src/**/*.py
+    bears = SpaceConsistencyBear
+    use_spaces = True
 
-|Scrutinizer Code Quality| |codecov.io|
+Store this file in the project's root directory and run coala:
 
-|Documentation Status| |Gitmate|
+.. code-block:: bash
 
-Newcomers Guide and Getting Involved
-------------------------------------
+    $ coala
 
-If you are new and would like to contribute, read our `Getting Involved Site
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__!
+Please read our
+`coafile specification <http://coala.readthedocs.io/en/latest/Users/coafile.html>`__
+to learn more.
 
-We appreciate any help! Feel free to message us on
-`gitter <https://gitter.im/coala-analyzer/coala>`__. If you have any
-questions we're happy to help you!
+If you want to quickly test some settings without a ``.coafile``, you
+can run:
```

Comment:
try something like: ``However, if you wouldn't like to save your settings, you can also run coala with some arguemnts from the command line, such as:``

 $ coala --files=setup.py --bears=SpaceConsistencyBear -S use_spaces=True

``Note that this code does the same thing as the one above.``